File: 8 kyu/Return Negative/main.dart
num makeNegative(n) {
  return n > 0 ? -n : n;
}

num makeNegative(n) => -n.abs();

num makeNegative(n)=> !n.isNegative ? -n : n;

num makeNegative(n) => n > 0 ? -n : n;

File: 8 kyu/Return Negative/main.go
package kata

func MakeNegative(x int) int {
  if x >= 0 {
    return -x
  }
  return x
}

func MakeNegative(x int) int {
    if x > 0 {
        return -x
    } else {
        return x
    }
}

func MakeNegative(x int) int {
	if x > 0 {
	  x *= -1
	}
	return x
}


import "math"

func MakeNegative(x int) int {
  return -int(math.Abs(float64(x)))
}